src/events/README.md

```diff
@@ -0,0 +1,72 @@
+# Events
+
+A microservices-demo service that provides event streaming services.
+
+## API Spec
+
+See the API Spec [here](./api-spec/events.json)
+
+## Build
+
+### Native Go
+
+```shell
+go build -o ./app cmd/main.go
+```
+
+The result is a binary named `app`, in the current directory.
+
+#### Docker
+
+`docker build -t mushop/events .`
+
+`docker-compose build`
+
+## Run
+
+## Environment
+
+The events service requires several environment configurations in order to produce messages on an Oracle Cloud Infrastructure Stream.
+
+| Variable | Description | Default | Required |
+|---|---|---|---|
+| `TENANCY` | Tenancy OCID string |  | Y |
+| `USER_ID` | IAM User OCID string |  | Y |
+| `PRIVATE_KEY` | API Private Key (pem) |  | Y |
+| `FINGERPRINT` | API Key Fingerprint  |  | Y |
+| `PASSPHRASE` | API Key passphrase  |  | N |
+| `REGION` | Oracle Cloud Infrastructure region (ex: `us-ashburn-1`)  |  | Y |
+| `STREAM_ID` | Stream OCID string |  | Y |
```

Comment:
when set the k8s secret to consume stream_id and message_endpoint, would be better to be separated from the other variables, to help the common usage of the oci service broker.

Reply:
> when set the k8s secret to consume stream_id and message_endpoint, would be better to be separated from the other variables, to help the common usage of the oci service broker.

Yes, that's the plan: `oci-credentials` + `oss-binding`.  Once OSB supports the streamPoolId, then it can just use kafka compat with secrets: OSB binding + auth token